### PR TITLE
test: add simple test for the base runner

### PR DESCRIPTION
**Pull request by ldorau**
This change is [Reviewable](https://reviewable.io/reviews/pmem/rpma/1440)

### CODE REVIEW

**Review by haichangsi — COMMENTED** [review]
> Reviewed all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1440)* status: 0 of 2 files reviewed, all discussions resolved (waiting on @haichangsi)

File: tools/perf/tests/lib/benchmark/conftest.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 
 ONESERIES_DUMMY = {'tool': 'dummy', 'mode': 'dummy', 'filetype': 'malloc', 'id': 'di'}
```
Comment (krzycz) [inline tools/perf/tests/lib/benchmark/conftest.py:13]: xxx

**Review by grom72 — COMMENTED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1440)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ldorau)

**Review by grom72 — COMMENTED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1440#-:-MrNiqMB6T-Yf5X-1gzs:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ldorau)

**Review by osalyk — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/pmem/rpma/1440#-:-MrRs-fF772DOYvA0pUX:bnfp4nl)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @ldorau)

**Pull request merged**